src/utils.ts

```diff
@@ -278,10 +273,8 @@ export function attachRenameType(setting: AttachmentPathSettings): ATTACHMENT_RE
     } else {
       ret = ATTACHMENT_RENAME_TYPE.ATTACHMENT_RENAME_TYPE_FILE;
     }
-  } else {
-    if (setting.attachmentPath.includes(SETTINGS_VARIABLES_NOTENAME) || setting.attachmentPath.includes(SETTINGS_VARIABLES_NOTEPATH)) {
+  } else if (setting.attachmentPath.includes(SETTINGS_VARIABLES_NOTENAME) || setting.attachmentPath.includes(SETTINGS_VARIABLES_NOTEPATH) || setting.attachmentPath.includes(SETTINGS_VARIABLES_PARENTFILE)) {
```

Comment:
Add `setting.attachmentPath.includes(SETTINGS_VARIABLES_PARENTFILE)` as the condition of  LINE271 
`if (setting.attachmentPath.includes(SETTINGS_VARIABLES_NOTENAME) || setting.attachmentPath.includes(SETTINGS_VARIABLES_NOTEPATH))`

->

`if (setting.attachmentPath.includes(SETTINGS_VARIABLES_NOTENAME) || setting.attachmentPath.includes(SETTINGS_VARIABLES_NOTEPATH) ||
setting.attachmentPath.includes(SETTINGS_VARIABLES_PARENTFILE))`